content/integrate/redis-data-integration/installation/install-k8s.md

```diff
@@ -94,7 +94,7 @@ To pull images from a local registry, you must provide the image pull secret and
 1.  Decompress the tar file:
 
     ```bash
-	tar -xvf rdi-k8s-<rdi-tag>.tar.gz 
+	tar -xvf rdi-<rdi-tag>.tar.gz 
```

Comment:
It looks like a tab character was used before "tar". I suggest replacing this with spaces. Here's an octal dump of the above snippet:

```
0000000    1   .           D   e   c   o   m   p   r   e   s   s       t
0000020    h   e       t   a   r       f   i   l   e   :  \n  \n        
0000040            `   `   `   b   a   s   h  \n  \t   t   a   r       -
0000060    x   v   f       r   d   i   -   <   r   d   i   -   t   a   g
0000100    >   .   t   a   r   .   g   z      \n                   `   `
0000120    `                                                            
0000121
```

In VS Code, the tar line does not line up with the indent of the line above.